app/models/playlist.rb

```diff
@@ -0,0 +1,11 @@
+class Playlist < ApplicationRecord
```

Comment:
D'ailleurs j'y pense: il faudrait vérifier que `genre` est un genre valide en fonction de la liste fournie par Spotify. Je mettrai la liste en dur mais je vérifierai quand même.

Reply:
Ce commentaire est toujours valide

Reply:
Yes, c'est noté,  j'attendais d'en être à l'API pour checker ce point !

Reply:
J'en mettrais quand même histoire de. Dans ta vue tu spécifies une liste finie (`genre1`, `genre2` et `genre3`), tu peux au moins valider que c'est l'un des 3, quitte à le modifier plus tard.